app/modules/import-export/assets/js/pages/import/import-kit/import-kit.js

```diff
@@ -128,11 +139,7 @@ export default function ImportKit() {
 					secondaryText={ __( 'Or', 'elementor' ) }
 					filetypes={ [ 'zip' ] }
 					onFileChoose={ () => eventTracking( 'kit-library/choose-file' ) }
-					onFileSelect={ ( file, e ) => {
-						setIsLoading( true );
-						importContext.dispatch( { type: 'SET_FILE', payload: file } );
-						eventTracking( 'kit-library/file-upload', null, 'feedback', null, null, e.type );
-					} }
+					onFileSelect={ ( file, e ) => runAction( file, e ) }
```

Comment:
```suggestion
					onFileSelect={ ( file, e ) => uploadFile( file, e ) }
```

Reply:
Feels it can be confusing, `runAction` can open the modal or uploading the file

Reply:
Done